internal/store/storageclass_test.go

```diff
@@ -59,15 +50,17 @@ func TestStorageClassStore(t *testing.T) {
 		{
 			Obj: &storagev1.StorageClass{
 				ObjectMeta: metav1.ObjectMeta{
-					Name:              "test_kube_storageclass_created",
+					Name:              "test_kube_storageclass-created",
```

Comment:
This was causing problems with the new regex :D. Also it's now consistent with the rest of the naming (see metric test above).